### PR TITLE
Install and restricted install on power

### PR DESCRIPTION
**Pull request by alishaIBM**
FYI : @vikram-redhat

### CODE REVIEW

**Comment by vikram-redhat** [comment]
@alishaIBM Hi - thanks for submitting this. Is this for the 4.7 release? Has this been reviewed by your QE team?

@SNiemann15

**Comment by lsmcfadden** [comment]
@vikram-redhat I think @alishaIBM meant to put WIP on this PR.  The idea is for her to do the PR and then the QE team can review and comment here.  Is this process okay?  If not, let me know and we'll conform to the RH norms.

In answer to your question, QE review is pending.

**Comment by vikram-redhat** [comment]
@lsmcfadden yes that is ok.

File: modules/installation-requirements-user-infra.adoc
Comment (SNiemann15) [inline modules/installation-requirements-user-infra.adoc:206]: IBM POWER8 or POWER9 processor-based systems 

@lsmcfadden Looking at the Power Guidelines (page 4) I think it should be written like this. However, I'm not the Power expert please confirm.

File: modules/installation-requirements-user-infra.adoc
Comment (SNiemann15) [inline modules/installation-requirements-user-infra.adoc:211]: * 6 IBM Power bare-metal servers or ...

File: modules/installation-requirements-user-infra.adoc
Comment (SNiemann15) [inline modules/installation-requirements-user-infra.adoc:218]: On your IBM Power instance, set up

@lsmcfadden Please confirm

File: modules/installation-requirements-user-infra.adoc
Comment (SNiemann15) [inline modules/installation-requirements-user-infra.adoc:239]: Delete "s" in systems

== Preferred IBM Power system requirements

File: modules/installation-requirements-user-infra.adoc
Comment (SNiemann15) [inline modules/installation-requirements-user-infra.adoc:243]: 6 IBM Power bare-metal...

File: modules/installation-requirements-user-infra.adoc
Comment (SNiemann15) [inline modules/installation-requirements-user-infra.adoc:248]: change as above

**Comment by lsmcfadden** [comment]
Yes
**IBM Power Systems** for the system 
**POWER** for CPU arch

File: modules/installation-requirements-user-infra.adoc
Comment (pravin-dsilva) [inline modules/installation-requirements-user-infra.adoc:222]: Only 1 bootstrap machine is required.

File: modules/installation-requirements-user-infra.adoc
Comment (pravin-dsilva) [inline modules/installation-requirements-user-infra.adoc:254]: Only 1 bootstrap machine is required for installation.

File: modules/installation-requirements-user-infra.adoc
Comment (arcprabh) [inline modules/installation-requirements-user-infra.adoc:259]: FWIK, DASD is supported on z. Also this line refers to zHPF but the header says POWER. Can you please recheck the content here @ktania46

File: modules/installation-requirements-user-infra.adoc
Comment (arcprabh) [inline modules/installation-requirements-user-infra.adoc:270]: Is this the min config or recommended config @alishaIBM

File: modules/installation-requirements-user-infra.adoc
Comment (ktania46) [reply modules/installation-requirements-user-infra.adoc:259]: Can you provide what is Power substitute for DASD, or who can provide information on this ? @arcprabh

File: modules/installation-requirements-user-infra.adoc
Comment (alishaIBM) [reply modules/installation-requirements-user-infra.adoc:270]: It is min config for ibm-power.

Comment (alishaIBM) [reply modules/installation-requirements-user-infra.adoc:270]: @arcprabh

File: modules/installation-requirements-user-infra.adoc
Comment (arcprabh) [inline modules/installation-requirements-user-infra.adoc:230]: This is 32Gb for powervm for masters.

File: modules/installation-requirements-user-infra.adoc
Comment (arcprabh) [reply modules/installation-requirements-user-infra.adoc:224]: @manojnkumar - pls check my comment and suggest the right term if we can use DASD for power.

**Comment by Prashanth684** [comment]
@arcprabh @manojnkumar will there be a section explaining multi-path as a day2 conifguration?

**Review by manojnkumar — APPROVED** [review]
/LGTM

File: modules/installation-requirements-user-infra.adoc
Comment (bergerhoffer) [inline modules/installation-requirements-user-infra.adoc:217]: systems is plural but it says "one instance". Should it be "One instance of an IBM POWER8 or POWER9 processor-based systems." ?

File: modules/installation-requirements-user-infra.adoc
Comment (bergerhoffer) [inline modules/installation-requirements-user-infra.adoc:228]: Need a space after "SSP" and before "(Shared Storage Pools)".

Also, does "Shared Storage Pools" need to be capitalized, or can it be "shared storage pools"?

File: modules/installation-requirements-user-infra.adoc
Comment (bergerhoffer) [inline modules/installation-requirements-user-infra.adoc:237]: Unless there's a reason that it needs to be capitalized, "Main Memory" should be sentence case to "main memory"

File: modules/installation-requirements-user-infra.adoc
Comment (bergerhoffer) [inline modules/installation-requirements-user-infra.adoc:254]: Same comment as above

File: modules/installation-requirements-user-infra.adoc
Comment (bergerhoffer) [inline modules/installation-requirements-user-infra.adoc:274]: Same comment as above

File: modules/installation-requirements-user-infra.adoc
Comment (bergerhoffer) [inline modules/installation-requirements-user-infra.adoc:265]: Same comment as above

File: modules/installation-requirements-user-infra.adoc
Comment (bergerhoffer) [inline modules/installation-requirements-user-infra.adoc:203]: I didn't realize until after reading through this that there was a recommended requirements section too. I feel like people might come to this section first and then stop before noticing that there is a different section which is actually what we recommend.

I'd consider putting the recommended section first, before the minimum requirement section, for this reason.

Additionally, you should add an introductory sentence both the minimum and recommended sections, just to reiterate that one is the bare minimum requirements, and that the other is what is actually recommended (for better performance, or whatever the reason is).

File: modules/installation-requirements-user-infra.adoc
Comment (bergerhoffer) [inline modules/installation-requirements-user-infra.adoc:245]: This section doesn't have that same first sentence about using IBM POWER8 or POWER9. Should it be added here too?

File: modules/installation-requirements-user-infra.adoc
Comment (bergerhoffer) [inline modules/installation-requirements-user-infra.adoc:263]: The title for the above section called this "IBM Power" (not just "Power"). Should it be that way here too?

File: modules/installation-requirements-user-infra.adoc
Comment (bergerhoffer) [inline modules/installation-requirements-user-infra.adoc:276]: If I'm comparing the sections correctly, this storage/memory section is the only difference between the minimum and recommended sections.

If that's correct, you could consider just having one section for the minimum requirements, and then when you get to here, specify that 16GB is the minimum for control plane machines, but that 32GB is recommended. And 8GB is minimum for compute machines, but that 32GB is recommended.

**Review by bergerhoffer — COMMENTED** [review]
A few questions and comments!

File: modules/installation-requirements-user-infra.adoc
Comment (SNiemann15) [reply modules/installation-requirements-user-infra.adoc:236]: As discussed we will leave this to be consistent with the section in the IBM Z docs.

File: modules/installation-requirements-user-infra.adoc
Comment (alishaIBM) [reply modules/installation-requirements-user-infra.adoc:278]: Leaving this as it is in IBM Z docs for consistency

File: modules/installation-requirements-user-infra.adoc
Comment (alishaIBM) [reply modules/installation-requirements-user-infra.adoc:309]: Leaving this as it is in IBM Z docs for consistency.

File: modules/installation-requirements-user-infra.adoc
Comment (bergerhoffer) [inline modules/installation-requirements-user-infra.adoc:245]: Some of these bullets have periods and others don't. Suggest going through and removing all of them in the PR (since they're not full sentences)

**Review by bergerhoffer — COMMENTED** [review]
Just one last thing about fixing the periods for consistency, and once that is done then I can merge!

**Review by bergerhoffer — COMMENTED** [review]
LGTM now, merging!

**Pull request merged**
